Add reference to charts-repo-actions-demo (#281)
helm/chart-testing: README.md

* [ct list-changed](doc/ct_list-changed.md)
* [ct version](doc/ct_version.md)

> For more extensive how-to guide it is recommended to see:

Reviewer: I'd suggest to reword this:

```suggestion
For a more extensive how-to guide, please see:

```
